napytau/ingest/reader/file_reader.py

```diff
@@ -0,0 +1,21 @@
+from os.path import isfile
+from typing import List
+
+from napytau.ingest.reader.reader import Reader
+
+
+class FileReader(Reader[str]):
+    @staticmethod
+    def read_rows(file_path: str) -> List[str]:
+        if not isfile(file_path):
+            raise FileNotFoundError(f"File not found: {file_path}")
+
+        with open(file_path) as file:
+            rows = file.readlines()
+
+        return list(
+            map(
+                lambda row: str(row),
+                rows,
```

Comment:
redundant, as lines are already strings

Reply:
please read the documentation before correcting, lines are actually of type `AnyStr`.

Reply:
I am blind. Sorry about that